src/middleware/options/TelemetryHandlerOptions.ts

```diff
@@ -21,11 +20,12 @@ import { MiddlewareOptions } from "./IMiddlewareOptions";
  * @property {number} RETRY_HANDLER_ENABLED - The hexadecimal flag value for retry handler enabled
  * @property {number} AUTHENTICATION_HANDLER_ENABLED - The hexadecimal flag value for the authentication handler enabled
  */
+
 export enum FeatureUsageFlag {
-	NONE = 0x0,
-	REDIRECT_HANDLER_ENABLED = 0x1,
-	RETRY_HANDLER_ENABLED = 0x2,
-	AUTHENTICATION_HANDLER_ENABLED = 0x4,
+	NONE = 0x0, // eslint-disable-line @typescript-eslint/naming-convention
```

Comment:
Is there a way to define the ignore rule for multiple lines similar to C#?

Reply:
Replaced with a single line.